q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,278 @@
+# Proposal for Q2: Structure of Governance
+
+## Definitions
+
+This document describes three roles involved in governance:
+
+* "Core Team"
+
+* "Steering Council"
+
+* "Vote Administrator"
+
+A section on each follows.
+
+The term "Perl Rules of Governance" means the final governance document to be
+written and ratified by the Gov-Begin group.
+
+## The Core Team
+
+The Core Team are a group of trusted volunteers involved in the ongoing
+development of the Perl language and interpreter.  They are not required to be
+language developers or committers.
+
+References to specific votes are explained in the "Rules for Voting" section.
+
+### Powers
+
+The Core Team has authority over the rules and composition of Perl governance.
+
+They alone elect the Steering Council and have the power to remove Steering
+Council members.
+
+In concert with the Steering Council, they manage their own membership and
+have the power to modify the Perl Rules of Governance.
+
+The Core Team do not have any authority over parts of the Perl ecosystem
+unrelated to developing and releasing the language itself.  These include, but
+are not limited to:
+
+- The Perl Foundation
+
+- CPAN administration and CPAN authors
+
+- perl.org, metacpan.org, and other community-maintained websites and services
+
+- Perl conferences and events, except those sponsored directly by a
+  governing group
+
+- Perl-related intellectual property legally owned by third-parties,
+  except as allowed by applicable licenses or agreements.
+
+### Membership
+
+The initial Core Team members will be specified in the first draft of the Perl
+Rules of Governance.
+
+Any Core Team member may nominate someone to be added to the Core Team by
+sending the nomination to the Steering Council.  The Steering Council must
+approve or reject the nomination.  If approved, the Steering Council will
+organize a Membership Change Vote to ratify the addition.
+
+Core Team members should demonstrate:
+
+* A solid track record of being constructive and helpful
+* Significant contributions to the project's goals, in any form
+* Willingness to dedicate some time to improving Perl
+
+Contributions are not limited to code. Here is an incomplete list of areas
+where contributions may be considered for joining the Core Team:
+
+* Working on community management and outreach
+* Providing support on mailing lists, IRC, or other forums
+* Triaging tickets
+* Writing patches (code, docs, or tests)
+* Reviewing patches (code, docs, or tests)
+* Participating in design decisions
```

Comment:
Should this be "discussions" instead of "decisions"?

Reply:
Done.